docs/src/getting-started/first-steps.md

```diff
@@ -6,7 +6,7 @@ We demonstrate how to create a new project, compile, and test it.
 To start a new project with Starknet Foundry, clone the template repository
 
 ```shell
-$ git clone https://github.com/foundry-rs/starknet_forge_template.git
+git clone https://github.com/foundry-rs/starknet_forge_template.git
```

Comment:
This is a nitpick, but I believe in most cases we have `$` to indicate, that this should be run as normal user (in contrary with `#` for superuser). Perhaps we should be consistent, and either have `$` for every shell example, or for none?

Reply:
I think it is left this way so it can be copy-pasted to the terminal without removing the dollar sign manually. It makes sense for command snippets that are supposed to be copied 1:1 imo. In other cases - the dollar sign makes it more aesthetic, but that's just my opinion

Reply:
We use `$` in most cases and I agree it makes sense, it's just impractical to have it here because when copying the snippet to clipboard, it also includes the `$` sign.

Reply:
sure, I've no strong opinion about that :+1: